.gitignore

```diff
@@ -2,4 +2,4 @@
 *.pyc
 .DS_Store
 .directory
-/node_modules
\ No newline at end of file
+*node_modules
```

Comment:
Aren't there other node_modules folders still in the tree? I assume we only want to remove this individual test's (as long as the root one) for now

Reply:
good point - will adjust

Reply:
done 👍